lib/libzrepl/data_conn.c

```diff
@@ -81,6 +81,7 @@ zio_cmd_alloc(zvol_io_hdr_t *hdr, int fd)
 		zio_cmd->buf_len = hdr->len;
 	}
 
+	atomic_inc_64(&zinfo->zio_cmd_inflight);
```

Comment:
zio_cmd_inflight may mean different thing for those who have not looked in to code. Would be better if we rename if to zio_cmd_allocated.

Reply:
It was meant to represent allocated `zio_cmd` at the moment. `zio_cmd_allocated` might be confused with allocated `zio_cmd` till the moment. How about `zio_cmd_in_use`?

Reply:
'zio_cmd_in_use' looks to be good